### DISCUSSION TITLE
How can I set it to not display unnecessary snippets? Just like the behavior in VSCode. (I'm not quite sure how to describe it; I've included a screenshot.)

### DISCUSSION THREAD

**Discussion by gengdz** [saghen/blink.cmp#492] [Q&A] [answered]
in neovim
![image](https://github.com/user-attachments/assets/45cd700c-71a6-4e6d-9632-b5992173ba44)

in VSCode 
![image](https://github.com/user-attachments/assets/02b607aa-d9af-4798-8997-a31b75a8d0e5)

Some background information：
* `.tsx` file
* using LazyVim's `coding.blink` extra without any other configuration.
* [my configuration file](https://github.com/gengdz/dotfiles/blob/main/nvim/lua/plugins/coding.lua)

**I expect the effect to be the same as in VSCode**, ***do not display tags like `<uselistser>${1}</uselistser>`***. I don't know why such tags are being displayed. 

---

In fact, this tag displays for any character. I expect that if the HTML actually has this tag, then the snippet should be displayed; if it doesn't have this tag, then it should not be displayed.

in neovim

![image](https://github.com/user-attachments/assets/aafacddf-da0b-4e81-bbd8-8710b679fb4d)

in VSCode

![image](https://github.com/user-attachments/assets/5c3e8bdb-3105-49a4-87c8-0f1b88944715)

**Comment by saghen** [comment, marked as answer]
It looks like these are coming from your LSP, or something like `emmet`. I'd investigate your LSP configuration

**Reply by gengdz** [reply]
Thank you very much! I solved the problem following your idea.

By using `emmet-language-server` instead of `emmet-ls`.

This is a bug with `emmet-ls`, you can see https://github.com/aca/emmet-ls/issues/55
